fontbe/src/orchestration.rs

```diff
@@ -385,9 +385,37 @@ impl GvarFragment {
                     return None;
                 }
 
-                let tuple_builder = TupleBuilder::new(region, axis_order);
-                let (min, peak, max) = tuple_builder.build();
-                Some(GlyphDeltas::new(peak, deltas.clone(), Some((min, max))))
+                let mut mins = Vec::<F2Dot14>::new();
+                let mut peaks = Vec::<F2Dot14>::new();
+                let mut maxes = Vec::<F2Dot14>::new();
+
+                // Optimisation: only write explicit mins and maxes if at least one of the tents is
+                // an intermediate region.
+                // https://github.com/fonttools/fonttools/blob/b467579c/Lib/fontTools/ttLib/tables/TupleVariation.py#L184-L193
+                let mut needs_intermediate = false;
+
+                for tag in axis_order.iter() {
+                    let tent = region.get(tag).unwrap();
+                    let (min, peak, max) = (
+                        F2Dot14::from_f32(tent.min.to_f64() as _),
+                        F2Dot14::from_f32(tent.peak.to_f64() as _),
+                        F2Dot14::from_f32(tent.max.to_f64() as _),
+                    );
+
+                    mins.push(min);
+                    peaks.push(peak);
+                    maxes.push(max);
+
+                    // NOTE: Comparison must occur after type conversion.
```

Comment:
Nit: it would be more helpful to say comparison must occur in SomeType because Reason

Reply:
Good shout, I've now expanded on it